apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -129,6 +138,9 @@ class SettingsCog extends Component {
           showTail={true}
         >
           <ManageAssets onClick={this.manageAssets} />
+          {experiments.isEnabled('student-libraries') && (
```

Comment:
Can you pull the 'student-libraries' experiment string to a constant here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/experiments.js#L20

Reply:
✔️